waspc/data/Generator/templates/sdk/wasp/server/crud/_operationTypes.ts

```diff
@@ -37,7 +37,7 @@ type _WaspEntity = {= crud.entityUpper =}
 /**
  * PUBLIC API
  */
-export namespace {= crud.name =} {
+export declare namespace {= crud.name =} {
```

Comment:
Fixing a TS lint, namespaces and modules should not be mixed in the runtime time, so we make it only a type thing.

Reply:
Hm, can't get this error either. What threw it?

Reply:
Nothing, just the recommendations [on TypeScript's handbook](https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html). If we just do `export declare` there's no output, so no interplay between JS modules and namespaces.